README.rst

```diff
@@ -25,9 +26,15 @@ this worked on ubuntu 14.04:
 ``sudo apt-get install libglib2.0-dev libatlas-base-dev``
 and for the testsuite also ``sudo apt-get install libgsl0-dev``
 
-Install on Windows / OSX
+Install on OSX
+===============
+Library compiles on OSX (thus python interface works), however console interface doesn't.
```

Comment:
Please remove ``(thus python interface works)`` it's not relevant here. We can add this to the python wrapper README if it's useful.